com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeVolumePerSceneData.cs

```diff
@@ -167,6 +170,7 @@ void OnEnable()
         {
             ProbeReferenceVolume.instance.RegisterPerSceneData(this);
 
+            ResolveSharedCellData();
```

Comment:
If this is the only place where we do it, what happens with multiple bakes when we queue up the loading?

Reply:
At the end of baking there is already a call to resolvecells, which reloads all cell data

Reply:
Ok, should be fine worth pinging @Unity-Technologies/gfx-qa-hdrp  to hammer a bit the loading process in testing this PR (loading already baked and doing a bunch of bakes after changing the scene structure a bit)